src/Compilers/Core/MSBuildTask/Microsoft.Build.Tasks.CodeAnalysis.csproj

```diff
@@ -93,22 +93,22 @@
   </Target>
 
   <Target Name="GenerateCompilerVersionTargetsFile"
-          Inputs="$(MSBuildThisFileFullPath)"
           Outputs="$(CompilerVersionTargetsFileIntermediatePath)">
 
     <PropertyGroup>
-      <CompilerApiVersion>$([System.Version]::Parse($(VersionPrefix)).Major).$([System.Version]::Parse($(VersionPrefix)).Minor)</CompilerApiVersion>
-      <CompilerVersionTargetsFileContent>
+      <_CompilerApiVersion>$([System.Version]::Parse($(VersionPrefix)).Major).$([System.Version]::Parse($(VersionPrefix)).Minor)</_CompilerApiVersion>
```

Comment:
Used the `_` to make it clearer they are target local properties vs. accessible ones everywhere. This confused me a bit when I first started looking at it.